File: wls-ergebnismeldung-service/src/main/java/de/muenchen/oss/wahllokalsystem/ergebnismeldungservice/domain/common/BezirkUndWahlIDStapelart.java
package de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.domain.common;

import jakarta.persistence.Column;
import jakarta.persistence.EnumType;
import jakarta.persistence.Enumerated;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import java.io.Serial;
import java.io.Serializable;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class BezirkUndWahlIDStapelart implements Serializable {

    @Serial
    private static final long serialVersionUID = 1L;

    @NotNull
    @Size(max = 1024)
    @Column(name = "wahlbezirkID")
    private String wahlbezirkID;

    @NotNull
    @Size(max = 1024)
    @Column(name = "wahlID")
    private String wahlID;

    @Enumerated(EnumType.STRING)
    @NotNull
    private Stapelart stapelart;
}

File: wls-gui-wahllokalsystem/src/types/ergebnismeldung/common/BezirkUndWahlIDStapelArt.ts
import type { BezirkUndWahlID } from "@/types/ergebnismeldung/common/BezirkUndWahlID.ts";
import type { StapelArtEnum } from "@/types/ergebnismeldung/common/StapelArtEnum.ts";

export interface BezirkUndWahlIDStapelArt extends BezirkUndWahlID {
  stapelArt: StapelArtEnum;
}
